tests/test_c_contiguous.py

```diff
@@ -9,27 +9,19 @@ def test_c_contiguous_array_input(self):
         """
         Test whether cpp sides receives contiguous array
         """
-        spl_ts = (
-            c.splinepy.Bezier,
-            c.splinepy.RationalBezier,
-            c.splinepy.BSpline,
-            c.splinepy.NURBS,
-        )
-        spl_props = (c.z2P2D, c.r2P2D, c.b2P2D, c.n2P2D)
-
-        for spline_t, props in zip(spl_ts, spl_props):
-            p = props.copy()  # create shallow copy of the dict
 
+        for spl, props in zip(
+            c.get_all_spline_typs_as_list(),
+            c.get_all_splines_as_dict_as_list(),
+        ):
             # make f contiguous
-            f_contig_orig = c.np.asarray(p["control_points"], order="F")
+            f_contig_orig = c.np.asarray(props["control_points"], order="F")
```

Comment:
idk, in my head, props sounds like saying "proper respect", as in props, why not just use the entire word?
```suggestion
            f_contig_orig = c.np.asarray(properties["control_points"], order="F")
```